suites/pacific/rgw/tier-1-upgrade_4x_to_5x_latest.yaml

```diff
@@ -0,0 +1,412 @@
+#############################################################################
+# - Automation support for upgrade from RHCS4 to RHCS5 cluster with RGW tests
+#
+# Test steps:
+# ---------------------------------------------------------------------
+# - Deploy bare-metal Nautilus Ceph cluster using CDN RPMS with lvm osd scenario
+# - Run RGW tests
+# - Convert the cluster to containerized using ceph-ansible
+# - Upgrade to Pacific using ceph-ansible
+# - Run RGW tests after upgrade
+#
+#############################################################################
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: True
+
+- test:
+    name: ceph ansible install rhcs 4.x from cdn
+    polarion-id: CEPH-83573588
+    module: test_ansible.py
+    config:
+      use_cdn: True
+      build: '4.x'
+      ansi_config:
+        ceph_origin: repository
+        ceph_repository: rhcs
+        ceph_repository_type: cdn
+        ceph_rhcs_version: 4
+        ceph_stable_release: nautilus
```

Comment:
This has no impact as the `ceph_repository` is set to `rhcs`

Reply:
Done. Keeping only  `ceph_repository: rhcs`